mlflow/langchain/_rag_utils.py

```diff
@@ -0,0 +1,5 @@
+__chain__ = None
+
+
+def _set_chain(chain):
```

Comment:
Can we add a docstring to this method explaining why it's here / what it does / what the scope is? Can we do the same for `_set_config_path` once we move it here?